lib/pier_logging/request_logger.rb

```diff
@@ -44,7 +30,7 @@ def log(args)
       env, status, type, body, starts_at, ends_at, _ = args
       request = Rack::Request.new(env)
       request_headers = get_request_headers_from_env(env)
-      logger.info redact_object({
+      logger.info PierLogging::Helpers::Redactor.redact({
```

Comment:
Caso o logger definido for um `PierLogging::Logger`, esse `.redact` vai rodar 2 vezes, já que agora tudo que passar pelo `Logger` já é obfuscado pelo `PierLogging::Helpers::Redactor`.
Pode ter impacto na performance.

Reply:
Não entendi aqui @brunoarakaki todos os nossos logs passarão 2x pelo `.redact`?

Reply:
@banduk Os logs aqui no `RequestLogger` podem passar 2 vezes, uma aqui antes de enviar o log e depois de novo dentro `PierLogging::Logger#_log`.

## Primeiro redact
Aqui, antes de logar, estamos obfuscando os argumentos que serão enviados para o `logger`, com o `PierLogging::Helpers::Redactor.redact`.


## Segundo redact
O `logger` aqui nesse contexto, pode ser um `PierLogging::Logger` (mas pode não ser também, já que essa dependência está sendo injetada no `initialize`).

Com a mudança que fizemos, o método `PierLogging::Logger#info` já está obfuscando tudo que passamos para ele (por conta do override em `PierLogging::Logger#_log`.

Reply:
Removi esse redact extra, o logger base já tá cuidando disso.